### PR TITLE
Beta 1 release

### PR DESCRIPTION
**Pull request by gsanchietti**
https://trello.com/c/60oH89Ez/353-release

Thanks also to https://open-assistant.io

Commits should be squashed on merge

### CODE REVIEW

File: release_notes.rst [hunk 1]
```diff
@@ -2,24 +2,63 @@
...
+Releases glossary
+=================
+
+The software release cycle includes four stages: Alpha, Beta, Release Candidate (RC), and Stable.
+
+During the **Alpha** stage, the software is not thoroughly tested and may not include all planned features.
+This release is not suitable for production environments. However, it can be used to preview what's coming in the upcoming version.
+Please note that updates from an Alpha release to other releases are not supported.
+
+The **Beta** stage indicates that the software is feature complete, but it may still contain many known and unknown bugs.
```
[the remark this answers is not in the document]
Comment (gsanchietti) [reply release_notes.rst:54]: I prefer the original form, even wikipedia says the same: https://en.wikipedia.org/wiki/Software_release_life_cycle#Beta

[the remark this answers is not in the document]
Comment (gsanchietti) [reply release_notes.rst:54]: We Googledup! :D

File: release_notes.rst [hunk 2]
```diff
@@ -2,24 +2,63 @@
...
+
+- Collaborative tools: includes Dovecot/Postfix/Rspamd mail server, WebTop, Roundcubemail, Nextcloud, Collabora Online, Dokuwiki, ejabberd, Mattermost
+- Development utilities: features MariaDB and NGINX web server for creating dynamic applications and services
+- Monitoring and analysis: offers Grafana, Prometheus, and node_exporter for tracking performance metrics and identifying potential issues
+- Data storage: uses MinIO for managing large amounts of structured and unstructured data
+- Network defense: implements CrowdSec for protecting local applications against remote attacks
+
+The following known limitations will be resolved in future updates:
+
+- remote removal of offline nodes from the cluster is currently unavailable
```
[the remark this answers is not in the document]
Comment (gsanchietti) [reply release_notes.rst:39]: If so, we can even close this one https://trello.com/c/5gtFDfCl/345-core-p-dead-node-module-removal

File: release_notes.rst [hunk 3]
```diff
@@ -2,24 +2,64 @@
...
+- Centralized logging: collect all logs into one place for easy monitoring
+- Configuration and data backups: regularly save cluster settings and data application to remote providers like Amazon S3 and Backblaze B2
+- Authentication: support for both Active Directory and LDAP (RFC2307) user directories
+- File server: features an SMB (Server Message Block) file server that enables seamless integration with Windows-based networks
+- Auditing: track changes made within the system to ensure security and accountability
+- Email relay: use a smart host to route outgoing email traffic through a trusted server
+- Custom web routing: define custom URLs to handle specific requests
+- Multi-factor authentication: enable two-step verification for administrator accounts
+- Built-in firewall: protect against unauthorized access at the network level by implementing a local firewall
+- Migration: :ref:`Cockpit module <migration-section>` to import NethServer 7 applications
```
[the remark this answers is not in the document]
Comment (gsanchietti) [reply release_notes.rst:28]: Mmm no: double repetition of "application" word.

File: release_notes.rst [hunk 4]
```diff
@@ -2,24 +2,63 @@
...
-  Do not use this release on production. You can use this release to test the software before using it
-  on production environments.
-  Updates from beta release to RC and stable are supported but may require manual procedure.
+**Beta 1** release.
 
-* **Release Candidate (RC)**: the software is feature complete, it contains no known bugs and can be
-  promoted to stable if no major issues arise.
-  Updates from RC to stable is supported and should be almost automatic.
-  Use this release on production only if you already have some experience with the software.
+Main core features includes:
```
Comment (filippocarletti) [inline release_notes.rst:16]:
```suggestion
Main core features include:
```

File: release_notes.rst [hunk 5]
```diff
@@ -2,24 +2,63 @@
...
 
-* **Stable**: the software can be safely used on production environments.
+- Node management: add and remove nodes from the system
+- Centralized logging: collect all logs into one place for easy monitoring
+- Configuration and data backups: regularly save cluster settings and data application to remote providers like Amazon S3 and Backblaze B2
```
Comment (filippocarletti) [inline release_notes.rst:20]:
```suggestion
- Configuration and data backups: regularly save cluster settings and application data to remote providers like Amazon S3 and Backblaze B2
```

File: release_notes.rst [hunk 6]
```diff
@@ -2,24 +2,63 @@
...
+The **Beta** stage indicates that the software is feature complete, but it may still contain many known and unknown bugs.
+This release should not be used on production environments. However, it can be used to test the software before deploying it to production.
+Updates from a Beta release to an RC or Stable release are supported but may require a manual procedure.
+
+During the **Release Candidate (RC)** stage, the software is feature complete, and it contains no known bugs.
+If no major issues arise, it can be promoted to Stable. Updates from an RC release to a Stable release are supported
+and should be almost automatic.
+However, if you're new to the software, it's best to use it on production only if you already have some experience with it.
+
+The **Stable** release is the most reliable and safe to use on production environments.
```
Comment (filippocarletti) [inline release_notes.rst:63]:
```suggestion
The **Stable** release is the most reliable and safe to use in production environments.
```

**Pull request merged**